BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java

```diff
@@ -79,6 +97,22 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return mainView;
     }
 
+    private int countFilesInDirectory(File dir) {
+        if (dir.exists() && dir.isDirectory()) {
+            File[] files = dir.listFiles();
+            if (files != null) {
```

Comment:
you could simplify this. (in a separate PR) 
var files = dir.listFiles(File::isFile);
return files == null ? 0 : files.length;